client/web/workflow/src/components/ExpressionEditor.vue

```diff
@@ -2,16 +2,13 @@
   <div
     class="position-relative"
   >
-    <ace-editor
+    <c-ace-editor
       v-model="expressionValue"
       :lang="lang"
-      :mode="lang"
-      theme="chrome"
-      width="100%"
       :height="height"
-      :class="{ 'border-0 rounded-0': !border }"
+      :show-line-numbers="showLineNumbers"
+      :font-size="fontSize"
       v-on="$listeners"
-      @init="editorInit"
     />
 
     <b-button
```

Comment:
As far as i know this (show popout) is now in c-ace editor, so you don't need this here anymore.
That means that <expression-editor, can be replaced with <c-ace-editor> Since <expression-editor> is just a wrapper for ace editor in workflows anyway.